modular_skyrat/master_files/code/modules/uplink/uplink_items.dm

```diff
@@ -4,123 +4,27 @@
 
 //Place any new uplink items in this file, and explain what they do
 
-//BUNDLES
-/* /datum/uplink_item/bundles_tc/spaceassassin
-	name = "Space Assassin Bundle"
-	desc = "A unique kit commonly used by military infiltrators and the like to get the drop on unsuspecting crew, perfect for the aspiring covert assassin and stealthy manipulator."
-	item = /obj/item/storage/box/syndie_kit/spaceassassin
-	cost = 30 //40 tc would have been better
-*/ //To be balanced
 //DANGEROUS
-/datum/uplink_item/dangerous/aps_traitor
-	name = "Stechkin APS Machine Pistol"
-	desc = "An ancient Soviet machine pistol, refurbished for the modern age. Uses 9mm auto rounds in 15-round magazines and is compatible \
-			with suppressors. The gun fires in three round bursts."
-	item = /obj/item/gun/ballistic/automatic/pistol/aps
-	cost = 10
-
 /datum/uplink_item/dangerous/foamsmg_traitor
 	name = "Toy Submachine Gun"
 	desc = "A fully-loaded Donksoft bullpup submachine gun that fires riot grade darts with a 20-round magazine."
 	item = /obj/item/gun/ballistic/automatic/c20r/toy/unrestricted/riot
 	cost = 5
 
-/datum/uplink_item/dangerous/revolver_alt
-	name = "Unica Six Revolver"
-	desc = "A retro high-powered autorevolver typically used by officers of the New Russia military. Uses .357 ammo."
-	item = /obj/item/gun/ballistic/revolver/mateba
-	cost = 13
-	surplus = 50
-
-/datum/uplink_item/dangerous/holocarp
```

Comment:
They are not a duplicate.

Reply:
there is missing context here from when we removed holocarp from the uplink years ago at /tg/ that is lost to time until i dive through the PR archives, but we removed holocarp specifically for being a holoparasite duplicate back when we made holoparasites not stackable anymore iirc

Reply:
if the holocarp is to remain, then it's getting properly bumped up to the same price and progression requirements as Holoparasites are